### PR TITLE
Minor Charter Corrections

### PR DESCRIPTION
**Pull request by LorbusChris**
Fix punctuation and semantics in a few places.
Thanks @jamescassell @cgruver @pecameron

WIP. A Call for Agreement will be made for this.

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -28,7 +29,7 @@ The OKD Working Group focuses on the following end-user related topics of the li
 * OKD delivery and release management
 * OKD management and operations
 
-The Working Group will work on developing best practices, fostering collaboration between related projects, and working on improving tool interoperability.  Additionally, the Working Group will propose new initiatives and projects when capability gaps in the current ecosystem are defined. 
+The Working Group will work on developing best practices, fostering collaboration between related projects, and working on improving tool interoperability. Additionally,the Working Group will propose new initiatives and projects when capability gaps in the current ecosystem are defined.
```
Comment (jamescassell) [inline CHARTER.md]: There should be a space after the comma.

File: CHARTER.md [hunk 2]
```diff
@@ -131,8 +132,8 @@ Additional Chairs may be added so long as the existing number of Chairs is odd.
 In the event that an even number of Chairs exist and vote situation arises, the Chairs will randomly select one chair to abstain. 
 
 ### Role of Sub-Working Groups 
-Each Sub-Working Group (SWG) must have a Chair working as an active sponsor. Under the mandate of the Working Group, each SWG will have the autonomy to establish their own charter, membership rules, meeting times, make in-scope decisions as delegated, and management processes.
+Each Sub-Working Group (SWG) must have a Chair working as an active sponsor. Under the mandate of the Working Group, each SWG will have the autonomy to establish their own charter, membership rules, meeting times, and management processes. Each SWG will also have the authority to make in-scope decisions as delegated by the Working Groups.
```
Comment (jamescassell) [inline CHARTER.md]: Singular working group at the end.

**Review by jamescassell — COMMENTED** [review]
Two typos, otherwise looks good to me.

**Comment by DanyC97** [comment]
@LorbusChris happy to approve it once you remove the hold , just buzz me when is ready

**Comment by LorbusChris** [comment]
Let's approve this tentatively in the next meeting (per the process defined on the charter)

File: CHARTER.md [hunk 3]
```diff
@@ -131,8 +132,8 @@ Additional Chairs may be added so long as the existing number of Chairs is odd.
...
+Each Sub-Working Group (SWG) must have a Chair working as an active sponsor. Under the mandate of the Working Group, each SWG will have the autonomy to establish their own charter, membership rules, meeting times, and management processes. Each SWG will also have the authority to make in-scope decisions as delegated by the Working Group.
 
-SWG are required to submit their agreed Charter to the Working Group for information and archival. The Chairs can petition for dissolution of an inactive or hostile SWG by Call for Agreement. Once dissolved the SWG’s delegated Charter and outstanding authority to make decisions is immediately revoked. The Chairs may then take any required action to restrict access to Working Group Resources.  
+SWG are required to submit their agreed Charter to the Working Group for information and archival. The Chairs can petition for dissolution of an inactive or hostile SWG by Call for Agreement. Once dissolved the SWG’s delegated Charter and outstanding authority to make decisions is immediately revoked. The Chairs may then take any required action to restrict access to Working Group Resources.
```
Comment (sgreene570) [inline CHARTER.md]: Should the first use of `SWG` be plural?

Comment (LorbusChris) [reply CHARTER.md]: yes, thanks!

**Review by Conan-Kudo — APPROVED** [review]

**Comment by LorbusChris** [comment]
approved in meeting
/hold cancel

**Pull request merged**